ovos_dinkum_listener/service.py

```diff
@@ -185,10 +185,11 @@ def state(self):
         return self._state
 
     def run(self):
-        """Service entry point"""
+        """
+        Service entry point
+        """
         try:
             self._state = ServiceState.NOT_STARTED
-            self._before_start()  # TODO: Is this ever necessary?
```

Comment:
why was this removed? that method needs to be called....

Reply:
It's called during init and I didn't see a case where it would need to be called again. I can replace this though, just appeared redundant